package.json

```diff
@@ -116,5 +133,18 @@
     "cross-env": "^5.1.1",
     "mock-require": "^2.0.2",
     "sinon": "^4.1.3"
+  },
+  "browserslist": {
+    "development": [
```

Comment:
Does this mean that every developer machine needs to have these browser versions available? How about macOS and linux users? (for edge)

Reply:
So, this was not about which browsers the browser tests uses but rather what compilation target Babel has when compiling.